src/index.html

```diff
@@ -16,7 +16,11 @@
       href="./styles/index.scss"
     />
```

Comment:
In the `href` attribute, you should be linking to a CSS file, not an SCSS file. SCSS files are preprocessed to generate CSS files. Make sure you are linking to the compiled CSS file, typically named `style.css` or `index.css`.